src/web/src/components/Posts/GitHubInfo.tsx

```diff
@@ -57,9 +57,10 @@ const filterGitHubUrls = (urls: string[]) => {
     // Ex: /Seneca-CDOT/telescope ✅
     // Ex: /Seneca-CDOT/telescope/pull/2367/commits/d3fagd3fagd3fagd3fagd3fagd3fag4d41265748 ✅
     // Ex: /Seneca-CDOT/telescope/issues ✅
-    const matches = /^\/(?<user>[^/]+)\/(?<repo>[^/]+)((\/(.*))?\/(?<type>[^/]+)\/(?<id>(\d+|\w{40}))\/?$)?/i.exec(
-      pathname
-    );
+    const matches =
```

Comment:
Unnecessary changes, revert this whole file.

Reply:
when pushing the new commit to GitHub, the system autotest. If I don't change it, it can't be uploaded.